src/hm/hmRadio.h

```diff
@@ -101,19 +104,20 @@ class HmRadio {
             // change the byte order of the DTU serial number and append the required 0x01 at the end
             DTU_RADIO_ID = ((uint64_t)(((dtuSn >> 24) & 0xFF) | ((dtuSn >> 8) & 0xFF00) | ((dtuSn << 8) & 0xFF0000) | ((dtuSn << 24) & 0xFF000000)) << 8) | 0x01;
 
+            SPIClass* spi;
             #ifdef ESP32
                 #if CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32S3
-                    mSpi = new SPIClass(FSPI);
+                    spi = new SPIClass(FSPI);
```

Comment:
das dürfen wir nicht machen, da wir `mSpi` an die NRF24 Klasse übergeben. Daher muss die `SPIClass` noch 'leben', auch zu späteren Aufrufen der NRF24-Lib

Reply:
Das hatte ich "blind" aus dem Vorschlag von beegee3 übernommen und kann das nicht qualifiziert beantworten.

Fakt ist: der Code liefert zumindest auf dem ESP32 in dieser Form sehr zuverlässig und schnell Antworten von den Invertern. 

@beegee3 :  Ist das wichtig, oder kann/soll ich das im PR zurückdrehen?